cron/eventStats.py

```diff
@@ -79,12 +80,16 @@ def main():
     logger.debug('starting')
     logger.debug(options)
     es = ElasticsearchClient((list('{0}'.format(s) for s in options.esservers)))
+    index = options.index
     stats = esSearch(es)
     logger.debug(json.dumps(stats))
     try:
-        # post to elastic search servers directly without going through
-        # message queues in case there is an availability issue
-        es.save_event(body=json.dumps(stats), doc_type='mozdefstats')
+        while es.index_exists(index) == False:
```

Comment:
I don't think we want this to run indefinitely like this, maybe we should have some sort of counter to keep track of how many times we've slept for, waiting for the index to get created? And after maybe the 3rd time, the script just exists with an error message saying the index isn't created?

Reply:
This change has been committed.